awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -434,15 +439,13 @@ def authenticate(self, **kwargs):
         # If we have neither of these, then we can try un-authenticated access
         self.authenticated = True
 
-    def default_check_mode(self):
```

Comment:
Was this already a defunct method?

Reply:
Yea, this was from way back in the beginning when I had assumed that every module would be dealing with check mode on its own. The thought was that anything that wanted more than a ping would override this function. But then we figured out how to deal with check mode internally to the class.